### ISSUE TITLE
Holograms (armor stands) are visible

### ISSUE THREAD

**Issue by Artuto** [ProtocolSupport/ProtocolSupportPocketIssues#55]
As seen here ![Meme](https://cdn.discordapp.com/attachments/286106780189065216/525340772287905792/Screenshot_2018-12-20-09-55-39.png) armor stands are completely visible. I'll test if this affects to every entity with invisibility.

**Comment by colinrgodsey** [comment]
Hm still need to figure out what causes this. Think its some kind of order-of-operations with how the flags are set, or maybe its set on initial spawn? Probably something silly.

**Comment by Artuto** [comment]
Yes, HolographicDisplays, however, if you relogin they're invisible, only happens on creation.

**Comment by colinrgodsey** [comment]
oh right, color changing effect

**Comment by colinrgodsey** [comment]
also curious if this still happens with the new changes
